scripts/kind-build-test.sh

```diff
@@ -249,6 +255,9 @@ kubectl -n ack-system set env deployment/ack-"$AWS_SERVICE"-controller \
 sleep 10
 echo "ok."
 
+# start a background job which will refresh AWS credentials periodically
+"$SCRIPTS_DIR"/rotate-aws-creds-in-kind.sh "$AWS_SERVICE" &
```

Comment:
Can we store the PID returned by this call in an env var, then conditionally clean it up in the `clean_up` method?
Also I would prefer if this was behind an environment variable, because I don't want it running when I use this locally.

Reply:
>"Can we store the PID returned by this call in an env var, then conditionally clean it up in the clean_up method?"

We are cleaning up all the bgtask started by this script in cleanup now. (both on exit and interrupt)

>"Also I would prefer if this was behind an environment variable, because I don't want it running when I use this locally."

It is needed even locally for running tests more than 1 hour. Why should this not run locally? What's the worry?

Reply:
Fair enough, even local running tests will require this. I am hesitant to have random threads being started with bash scripts, but I understand the necessity.